Orange/widgets/evaluate/owpredictions.py

```diff
@@ -442,15 +461,15 @@ def _commit_predictions(self):
             self.send("Predictions", None)
             return
 
-        class_var = self.class_var
-        if class_var and class_var.is_discrete:
+        if self.class_var and self.class_var.is_discrete:
             newmetas, newcolumns = self._classification_output_columns()
         else:
             newmetas, newcolumns = self._regression_output_columns()
 
         attrs = list(self.data.domain.attributes) if self.output_attrs else []
         metas = list(self.data.domain.metas) + newmetas
-        domain = Orange.data.Domain(attrs, class_var, metas=metas)
+        domain = \
+            Orange.data.Domain(attrs, self.data.domain.class_var, metas=metas)
```

Comment:
If we assure at the beginning that `self.data.domain.class_var` must be equal to `self.class_var`, then the second (shorter/direct) version would look nicer here. And the line break would probably not be necessary :)